jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/surfaces/CardTests.java

```diff
@@ -1,26 +1,63 @@
 package io.github.epam.material.tests.surfaces;
 
-import static io.github.com.StaticSite.cardFrame;
+import static io.github.com.MaterialNavigator.openSection;
 import static io.github.com.StaticSite.surfaceCardPage;
+import static io.github.com.pages.surfaces.CardPage.*;
+import static com.epam.jdi.light.asserts.core.SoftAssert.jdiAssert;
 
+import com.epam.jdi.tools.Timer;
 import io.github.epam.TestsInit;
+import org.hamcrest.Matchers;
+import org.testng.annotations.BeforeTest;
 import org.testng.annotations.Test;
 
 public class CardTests extends TestsInit {
 
-    private final String EXPECTED_CARD_TITLE = "Card Header";
-    private final String EXPECTED_CARD_SUB_HEADER = "Sub-header";
-    private final String EXPECTED_CARD_CONTENT = "Sample card";
+    @BeforeTest
+    public void beforeTest() {
+        openSection("Card");
+        surfaceCardPage.isOpened();
+    }
+
+    private void textCheck(int index) {
+        String expectedPTagText = "Word of the Day";
+        String expectedH2TagText = "be•nev•o•lent";
+        jdiAssert(pTagTexts.get(index).text(), Matchers.is(expectedPTagText));
+        jdiAssert(h2TagTexts.get(index).text(), Matchers.is(expectedH2TagText));
+    }
+
+    @Test
+    public void simpleCardTest() {
+        textCheck(1);
+    }
 
     @Test
-    public void defaultPaperTest() {
-        surfaceCardPage.open();
-        cardFrame.card.getCardButtons().get(1).is().enabled();
-        cardFrame.card.getCardHeaders().get(1).is().text(EXPECTED_CARD_TITLE);
-        cardFrame.card.getCardHeaders().get(2).is().text(EXPECTED_CARD_SUB_HEADER);
-        cardFrame.card.is().backgroundImageDisplayed();
-        cardFrame.card.is().backgroundClickable();
-        cardFrame.card.clickOnBackground();
-        cardFrame.card.getInnerCardContent().get(1).is().text(EXPECTED_CARD_CONTENT);
+    public void outlinedCardTest() {
+        textCheck(2);
+        String expectedClassName = "MuiPaper-outlined";
+        jdiAssert(outlinedCard.hasClass(expectedClassName), Matchers.is(true));
```

Comment:
Please put class name in jdiAssert without creating variable

Reply:
done